x-pack/plugins/apm/public/plugin.ts

```diff
@@ -86,19 +87,45 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       pluginSetupDeps.home.featureCatalogue.register(featureCatalogueEntry);
     }
 
-    // register observability nav
+    // register observability nav if user has access to plugin
     plugins.observability.navigation.registerSections(
-      of([
-        {
-          label: 'APM',
-          sortKey: 200,
-          entries: [
-            { label: 'Services', app: 'apm', path: '/services' },
-            { label: 'Traces', app: 'apm', path: '/traces' },
-            { label: 'Service Map', app: 'apm', path: '/service-map' },
-          ],
-        },
-      ])
+      from(core.getStartServices()).pipe(
+        map(([coreStart]) => {
+          if (coreStart.application.capabilities.apm.show) {
+            return [
+              // APM navigation
+              {
+                label: 'APM',
+                sortKey: 200,
+                entries: [
+                  { label: 'Services', app: 'apm', path: '/services' },
```

Comment:
Side note, we don't translate route labels?

Reply:
Fixed 👍